packages/fx-core/src/plugins/resource/simpleauth/plugin.ts

```diff
@@ -99,7 +99,7 @@ export class SimpleAuthPluginImpl {
       ctx
     );
 
-    DialogUtils.progressBar = ctx.ui?.createProgressBar(Constants.ProgressBar.provision.title, 3);
+    DialogUtils.progressBar = ctx.ui?.createProgressBar(Constants.ProgressBar.provision.title, 4);
```

Comment:
Can we get the number auto? If hard code does that mean each new added step needs to increase the number?

Reply:
Fixed. Thanks!